Common/src/config_structure.cpp

```diff
@@ -142,9 +147,9 @@ unsigned short CConfig::GetnZone(string val_mesh_filename, unsigned short val_fo
       strcpy (cstr, val_mesh_filename.c_str());
       mesh_file.open(cstr, ios::in);
       if (mesh_file.fail()) {
-        cout << "cstr=" << cstr << endl;
-        cout << "There is no geometry file (GetnZone))!" << endl;
-
+        if (rank == MASTER_NODE) {
+          cout << "There is no geometry file called " << cstr << "." << endl;
```

Comment:
No problem :-) ... As a side note, I have noticed that when you run the software in parallel the message output doesn't always  work. e.g. when you have an error in the config file. In other words, you need to run SU2_CFD (no parallel) to catch the errors...